pydash-front/src/registration/RegistrationPage.js

```diff
@@ -114,14 +127,42 @@ class RegistrationPage extends Component {
         e.preventDefault()
```

Comment:
This function now is ridiculously long. It does too much: 

- preventing normal form submission
- validation of required fields
- validation of email
- validation of password
- validation of password confirmation
- sending the request to the server
- checking the server's response
- showing success/error based on the server's response.

Try to split it up. :slightly_smiling_face:

Reply:
Oh, and the name `tryLogin` is obviously very weird for the registration page.

Reply:
Also done, similar to the previous one the functions are still a bit long but I feel they do their own thing and only that thing. Do you agree?